<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JetPrinterView</class>
 <widget class="QWidget" name="JetPrinterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Rychlost</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLineEdit" name="lineEditSpeed">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpeed">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>95</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>m/min</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>520</y>
     <width>300</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Tryska</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="pushButtonNozzleOpen">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="JetPrinterUI.qrc">
      <normaloff>:/images/box_open.ico</normaloff>:/images/box_open.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNozzleClose">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="JetPrinterUI.qrc">
      <normaloff>:/images/box_insert.ico</normaloff>:/images/box_insert.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>280</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Počítadlo</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLCDNumber" name="lcdNumber1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo1_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Product:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo1_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Celkem:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>300</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Tiskárna</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="pushButtonPwr">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="JetPrinterUI.qrc">
      <normaloff>:/images/shutdown.ico</normaloff>:/images/shutdown.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPwrStatus">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>labelPwrStatus</zorder>
   <zorder>pushButtonPwr</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>280</y>
     <width>300</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Info</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="labelInfo1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Stav tiskarny</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stav trysky</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Chyba</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stav krytu</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>520</y>
     <width>300</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Tisk</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="pushButtonPrintStart">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="JetPrinterUI.qrc">
      <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPrintStop">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="JetPrinterUI.qrc">
      <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>921</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Tisková úloha</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>605</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>200.000000000000000</width>
      <height>65.000000000000000</height>
     </rectf>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonScript2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>45</width>
      <height>50</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Odeslat do tiskarny&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="JetPrinterUI.qrc">
      <normaloff>:/images/Send.ico</normaloff>:/images/Send.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>609</x>
      <y>40</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditJob">
    <property name="geometry">
     <rect>
      <x>609</x>
      <y>210</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Úloha</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonScript1">
    <property name="geometry">
     <rect>
      <x>857</x>
      <y>160</y>
      <width>45</width>
      <height>50</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Odeslat do tiskarny&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="JetPrinterUI.qrc">
      <normaloff>:/images/Send.ico</normaloff>:/images/Send.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="JetPrinterUI.qrc"/>
 </resources>
 <connections/>
</ui>
